pygromos/files/blocks/imd_blocks.py

```diff
@@ -293,20 +293,20 @@ def __init__(self, RETL: bool=False, NRET: int=0, RET: List[float]=[],  LRESCALE
                  CONT: bool=False, content=None):
         super().__init__(used=True, content=content)
         if content is None:
```

Comment:
Should these actually be ints and not bools? The bools will be converted to 0/1 when written/read. If int is correct it should also be adjusted in the __init__()

Reply:
@candidechamp

Reply:
AH this problem lies here and the castings are nasty work arounds. ;) 
I recommend adapting the REPLICA parser, instead of changing the types.
See here:
https://github.com/rinikerlab/PyGromosTools/blob/4ebc4ffd8f4e738eb40d1cb103fc719b736e321a/pygromos/files/blocks/imd_blocks.py#L311

Reply:
@candidechamp @MTLehner